### PR TITLE
 Add adminConsoleURL to config.js

### PR DESCRIPTION
**Pull request by spadgett**
Bumps depedencies for https://github.com/openshift/api/pull/82

/assign @deads2k

### CODE REVIEW

**Comment by spadgett** [comment]
/hold

I still need to add validation

File: pkg/apis/webconsole/validation/validation.go [hunk 1]
```diff
@@ -58,6 +58,13 @@ func validateClusterInfo(config v1.ClusterInfo, fldPath *field.Path) ValidationR
 		validationResults.AddWarnings(field.Invalid(fldPath.Child("metricsPublicURL"), "", "required to view cluster metrics in the console"))
 	}
 
+	if len(config.AdminConsolePublicURL) > 0 {
+		_, urlErrs := ValidateURL(config.AdminConsolePublicURL, fldPath.Child("adminConsolePublicURL"))
```
Comment (deads2k) [inline pkg/apis/webconsole/validation/validation.go]: Is this one supposed to be `ValidateSecureURL`?

**Comment by deads2k** [comment]
bump lgtm

**Comment by spadgett** [comment]
Thanks, updated to use `ValidateSecureURL`

**Pull request merged**